src/components/Form.js

```diff
@@ -90,6 +93,17 @@ class Form extends React.Component {
         return this.props.formState.error || (typeof latestErrorMessage === 'string' ? latestErrorMessage : '');
     }
 
+    getFirstErroredInput() {
```

Comment:
Just come across this randomly but this is missing a JSDoc and should have an `@returns`